.github/ISSUE_TEMPLATE/developer-onboarding.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: Developer Onboarding
+about: Onboarding steps for developers.
+title: 'Developer Onboarding: GH_HANDLE'
+labels: Eng
+
+---
+
+# Developer Onboarding
+
+- Onboardee: _GH handle of person being onboarded_
+- Onboarder: _GH handle of onboard buddy_
+
+## Installation
+
+There are several tools we use locally that you will need to have.
+- [ ] Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) for local development.
+- [ ] [Install the cf CLI v7](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html#pkg-mac) for the ability to deploy
+
+## Access
+
+### Steps for the onboardee
+- [ ] Setup [commit signing in GitHub](https://github.com/weather-gov/wather.gov/blob/main/docs/dev/git-signing.md) and with git locally.
+- [ ] [Create a cloud.gov account](https://cloud.gov/docs/getting-started/accounts/)
+- [ ] Ensure you can login to your cloud.gov account via the CLI
+```bash
+cf login -a api.fr.cloud.gov  --sso
+```
+- [ ] Have an admin add you to cloud.gov org and set up your [sandbox developer space](#setting-up-developer-sandbox). Ensure you can deploy to your sandbox space.
+- [ ] Have an admin add you to our login.gov sandbox team (`.gov Registrar`) via the [dashboard](https://dashboard.int.identitysandbox.gov/).
```

Comment:
@loganmcdonald-noaa I reckon we don't need these two steps today, but maybe someday?

Reply:
The second will be different since we aren't using login.gov! So can remove that one.